docs/examples/golang/index.md

```diff
@@ -84,3 +84,97 @@ go run main.go
         "config": {
           "dual-primary-action": "action-none",
 ```
+
+## Subscription Example
+The following example reuses the libraries from the previous example, but instead performs a Subscribe. 
+It subscribes to the interface counters for all interfaces on the system to allow for real-time statistics
+to be collected. Wildcards are use to get updates for all interfaces.
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path"
+	"time"
+
+	"github.com/aristanetworks/goarista/gnmi"
+	pb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+var cfg = &gnmi.Config{
+    Addr:     "10.20.30.67:6030",
+    Username: "ansible",
+    Password: "ansible",
+}
```

Comment:
it's a little bit of add'l overhead, but we should change these to be flags/args.  then folks can just point these at DUTs that use their addresses.